ansible/provision-and-execute-tests/playbook.yml

```diff
@@ -102,15 +106,36 @@
 - name: Provision Windows test executor instances and execute tests
   hosts: windows
   vars:
-    node_version: 16.3.0
+    node_version: 16.14.0
```

Comment:
Can you also upgrade it in the Linux block as well? We should have the same version in both, ideally.